engines/dradis-api/app/controllers/dradis/ce/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Dradis::CE::API
+  module V1
+    class CommentsController < Dradis::CE::API::V1::ProjectScopedController
+      before_action :set_comment, only: [:show, :update, :destroy]
+      before_action :set_commentable, only: [:index, :create]
+
+      def index
+        @comments = @commentable.comments
+      end
+
+      def show; end
+
+      def create
+      end
+
+      def update
+      end
+
+      def destroy
+      end
+
+      private
+
+      def set_commentable
```

Comment:
This complexity is caused by the fact that we route the comments list to the same controller. ex: 

```
/api/evidence/:evidence_id/comments
/api/issues/:issue_id/comments
/api/notes/:note_id/comments
```
Let me know if there's an simpler way of doing this.